packages/calypso-ui/src/text-control/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import { TextControl as BaseComponent, withFocusOutside } from '@wordpress/components';
+import classNames from 'classnames';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class TextControl extends PureComponent {
+	constructor( props ) {
```

Comment:
There's a simpler form here. You can use 
```
this.state = { 
  isFocused: false
};
```

in the root of the component and babel will auto-generate the right constructor.

Reply:
Thank you for the tip here! Updated.